pkg/template/helper.go

```diff
@@ -69,45 +69,45 @@ func AddTestCases(cluster *factory.Cluster, names []string) ([]testCase, error)
 		"TestInternodeConnectivityMixedOS": func(applyWorkload, deleteWorkload bool) {
 			testcase.TestInternodeConnectivityMixedOS(cluster, applyWorkload, deleteWorkload)
 		},
-		"TestSonobuoyMixedOS": func(applyWorkload, deleteWorkload bool) {
+		"TestSonobuoyMixedOS": func(_, deleteWorkload bool) {
```

Comment:
on here honestly i would not remove that, cause right now its showing us the  "conversion" to the expected signature

Reply:
the expected signature of the template (not the test itself) thats why the conversion

Reply:
Updated